app/services/c100_app/other_parties_decision_tree.rb

```diff
@@ -12,7 +12,7 @@ def destination
         after_personal_details(age_check: false)
       when :relationship
         children_relationships
-      when :contact_details
+      when :address_details
         after_contact_details
```

Comment:
Let's, for consistency, rename this from `after_contact_details` to `after_address_details`